### ISSUE TITLE
Current-page link is not programmatically labelled as such

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15333]
Current-page link is not programmatically labelled as such

-   **Severity:** Medium
-   **Affected Populations:**
    -   Low-Vision
    -   Cognitively Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -   Output Markup

#### Issue description

When a page is created and users view it, its name appears in the
sidebar under "Recent Posts", however there is no visual or semantic
indication that this link relates to the current page.

Since it's possible for authors to create multiple articles with the
same name, the lack of indication may cause confusion for users with
cognitive disabilities and screen reader users.

##### Issue Code

```
    <li>


        <a href="...">...</a>


    </li>
```

#### Remediation Guidance

Whenever the current page is reflected in a menu on that page, add
`aria-current="page"` to the applicable link.

Use the `aria-current` attribute as a hook to add a distinctive visual
style to separate it from other links.

##### Recommended Code

```
    <li>


        <a href="..." aria-current="page">...</a>


    </li>


    ...


    widget_recent_entries a[aria-current] {


        /* styles */


    }
```

#### Relevant standards

-   [2.4.4 Link Purpose (In Context)
    (Level A)](https://www.w3.org/TR/WCAG20/#navigation-mechanisms-refs)
-   [2.4.8 Location (Level
    AAA)](https://www.w3.org/TR/WCAG20/#navigation-mechanisms-location)

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-105 in Tenon's report

**Comment by afercia** [comment]
Looks like this is related to the Recent Posts widget, thus should be moved to the core Trac. Also, worth checking the output of the Pages, Archives, Categories, and Navigation Menu widgets.

**Comment by audrasjb** [comment]
Thanks for the issue. For your information I moved the issue to WordPress Core Trac: https://core.trac.wordpress.org/ticket/47094

**Comment by melchoyce** [comment]
Screenshot from the full report:

![image](https://user-images.githubusercontent.com/2846578/57185636-3e365f80-6e84-11e9-9be2-2a3e469758bf.png)

**Comment by tellthemachines** [comment]
Given that [the corresponding Trac ticket](https://core.trac.wordpress.org/ticket/47094) has been fixed, is it OK to close this one?

**Comment by afercia** [comment]
Yup, can be closed. Fixed in WordPress core in https://core.trac.wordpress.org/changeset/46163 for the 5.3 release.
